_translations/index_fi.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: signed
+title: An open letter in support of Richard M. Stallman
+description: An open letter in support of Richard Matthew Stallman being reinstated by the Free Software Foundation
```

Comment:
I'd appreciate it if you translate these as well.

Reply:
> I'd appreciate it if you translate these as well.

The page title and description have been translated to Finnish